### PR TITLE
fix: footer section responsiveness

### LINKED ISSUE #268: Footer Section not aligned
The footer section is not responsive on mobile. 

footer

### PR DESCRIPTION
**Pull request by shauryag2002**
fixes #268 

**Description**

- Fixes the responsiveness of footer section in mobile

**Video of changes:**

https://github.com/asyncapi/conference-website/assets/77102885/4c120b81-b9b6-4b46-8bc5-2f957e6927d0

### CODE REVIEW

**Comment by AceTheCreator** [comment]
@shauryag2002 Thanks for your contribution. Just a few suggested changes:

- On a smaller screen, you should make the footer section direction a column
- Make the code of conduct link readable

**Comment by RamGoel** [comment]
image
@shauryag2002 You can try making it align like this on mobile devices.

File: components/Footer/footer.js [hunk 1]
```diff
@@ -20,22 +20,22 @@ function Footer() {
...
+            href="https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md"
+            target="_blank"
+            rel="noreferrer"
+            className="hover:text-sky-500 duration-200 ease-in-out"
+          >
+            Code of Conduct
+          </a>
+        </div>
+        <div>
+        </div>
```
Comment (RamGoel) [inline components/Footer/footer.js:36]: suggestion: remove the indentation changes.

**Comment by RamGoel** [comment]
Looks good now, thanks @shauryag2002. @AceTheCreator you can give it a final look and merge

File: components/Footer/footer.js (hunk 1 above)
Comment (shauryag2002) [reply components/Footer/footer.js]: @RamGoel PTAL!

**Comment by AceTheCreator** [comment]
LGTM!!!

**Pull request merged**